tests/smoke/smoke_test.go

```diff
@@ -100,11 +100,12 @@ var _ = Describe("Elemental Smoke tests", func() {
 			Expect(out).Should(ContainSubstring("UTC"))
 		})
 
-		It("has default localectl configuration from cloud-init", func() {
+		// locale setting doesn't work on Fedora ¯\_(ツ)_/¯
+		/*It("has default localectl configuration from cloud-init", func() {
```

Comment:
So a bug in Fedora makes us disable this test for all other distros ? 🤔

Reply:
Not sure if a bug in Fedora... but yes that can be improved

Reply:
Re-enabled